build.sbt

```diff
@@ -20,8 +20,7 @@ import java.io.File
 // ============================================================================
 
 val scalacVersion = "2.13.11"
-// Since the release of GraalVM 23.0.0, the versioning is the same for Graal and OpenJDK.
-val graalVersion = "17.0.7"
+val graalVersion  = "17.0.7"
```

Comment:
Note that the newest Graal (PR is at #7991) does not have minor or patch version. It is just `21`:
```
> java -version
openjdk version "21" 2023-09-19
OpenJDK Runtime Environment GraalVM CE 21+35.1 (build 21+35-jvmci-23.1-b15)
OpenJDK 64-Bit Server VM GraalVM CE 21+35.1 (build 21+35-jvmci-23.1-b15, mixed mode, sharing)
```